src/org/omegat/gui/main/MainWindowMenuHandler.java

```diff
@@ -760,7 +764,7 @@ public void gotoEditorPanelMenuItemActionPerformed() {
      */
     public void gotoSegmentMenuItemActionPerformed() {
```

Comment:
We can unify all the action handler to take ActionEvent object.
It makes things simpler, even a patch become large.